core.go

```diff
@@ -555,6 +557,82 @@ func CountNonZero(src Mat) int {
 	return int(C.Mat_CountNonZero(src.p))
 }
 
+// CompleteSymm Copies the lower or the upper half of a square matrix to its another half.
+//
+// For further details, please see:
+// https://docs.opencv.org/3.4.0/d2/de8/group__core__array.html#gaa9d88dcd0e54b6d1af38d41f2a3e3d25
+//
+func CompleteSymm(m Mat, lowerToUpper bool) {
+	C.Mat_CompleteSymm(m.p, C.bool(lowerToUpper))
+}
+
+// ConvertScaleAbs Scales, calculates absolute values, and converts the result to 8-bit.
+//
+// For further details, please see:
+// https://docs.opencv.org/3.4.0/d2/de8/group__core__array.html#ga3460e9c9f37b563ab9dd550c4d8c4e7d
+//
+func ConvertScaleAbs(src Mat, dst Mat, alpha float64, beta float64) {
+	C.Mat_ConvertScaleAbs(src.p, dst.p, C.double(alpha), C.double(beta))
+}
+
+// CopyMakeBorder Forms a border around an image.
+//
+// For further details, please see:
+// https://docs.opencv.org/3.4.0/d2/de8/group__core__array.html#ga2ac1049c2c3dd25c2b41bffe17658a36
+//
+func CopyMakeBorder(src Mat, dst Mat, top int, bottom int, left int, right int, borderType int, value color.RGBA) {
+
+	c_value := C.struct_Scalar{
+		val1: C.double(value.B),
+		val2: C.double(value.G),
+		val3: C.double(value.R),
+		val4: C.double(value.A),
+	}
+
+	C.Mat_CopyMakeBorder(src.p, dst.p, C.int(top), C.int(bottom), C.int(left), C.int(right), C.int(borderType), c_value)
+}
+
+type DftFlags int
+
+const (
+	DftForward DftFlags = 0
```

Comment:
We should have doc for the flags, that would be very useful.

Reply:
indeed.